### PR TITLE
Bandh design refresh

### PR DESCRIPTION
**Pull request by karloscastaneda**

### CODE REVIEW

File: bandh/layouts/partials/home-banner.html [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+     }
+     else if (imgArray[rand] == imgArray[1]) {
+     	anchorPath.setAttribute('href', '/c/Wacom-TV/68963353--');
+     	anchorPath.setAttribute('title', 'Wacom-TV');
+     }
+     else{
+     	anchorPath.setAttribute('href', '/c/Seagate-TV/68963355--');
+     	anchorPath.setAttribute('title', 'Seagate-TV');
+     }
+
```
Comment (rfornes) [inline bandh/layouts/partials/home-banner.html]: This logic is needs to be in main.js

Comment (rfornes) [inline bandh/layouts/partials/home-banner.html]: Why vanilla JS? in main.js you have jQuery & other goodies

Comment (rfornes) [inline bandh/layouts/partials/home-banner.html]: Logic can be improved, [Look](https://jsfiddle.net/9tvv5zpm/)

Comment (rfornes) [inline bandh/layouts/partials/home-banner.html]: Also, we can't hardcode the link & other data, this shall be coming from our .md files (or theme.toml) we willfigured this out, priority is above comments.

Comment (karloscastaneda) [inline bandh/layouts/partials/home-banner.html]: got it, will fix it ASAP

File: bandh/static/js/main.js [hunk 2]
```diff
@@ -1804,33 +1836,34 @@
                         $.ajax({
                             url: "//app.tvpage.com/api/channels/"+channelId+"/videos",
                             dataType: 'jsonp',
+                            async:false,
```
Comment (rfornes) [inline bandh/static/js/main.js]: why do you need this to be synchronous?

Comment (karloscastaneda) [inline bandh/static/js/main.js]: because it was clonning the video grid, and loading the same videos over and over

**Pull request closed without merging**